app/models/user.rb

```diff
@@ -16,7 +16,7 @@ class User < ApplicationRecord
   has_many   :miq_widget_sets, :as => :owner, :dependent => :destroy
   has_many   :miq_reports, :dependent => :nullify
   has_many   :service_orders, :dependent => :nullify
-  has_many   :shares
+  has_many   :owned_shares, :class_name => "Share"
```

Comment:
@kbrock I'm not sure what you're asking.  Why would we create a has_many association we wouldn't use?  Can you clarify what you're concerned with?